02-IDE_exercises/solutions/08a-project/count_letters.py

```diff
@@ -3,23 +3,29 @@
 # Description: creates a function called count_letters that takes as a parameter a string
 #              and returns a dictionary that tabulates how many of each letter is in that string.
 
-def count_letters(string):
+def count_letters(string: str) -> dict[str, int]:
```

Comment:
incorrect type hint for dictionary. us the `from typing, import Dict` and then `Dict[str, int]`